core/src/main/java/amino/run/kernel/client/KernelClient.java

```diff
@@ -154,4 +299,86 @@ public void copyObjectToServer(InetSocketAddress host, KernelOID oid, KernelObje
                     MicroServiceReplicaNotFoundException {
         getServer(host).copyKernelObject(oid, object);
     }
+
+    /**
+     * Method to measure the kernel server metrics with respect to other remote servers and store
+     * them in {@link amino.run.kernel.client.KernelClient#metrics}. These collected metrics are
+     * reported to OMS in heartBeats
+     */
+    private void measureMetrics(final KernelServerInfo serverInfo) {
+        KernelServer server = serverInfo.remoteRef;
+        int dataLength = serverInfo.data.len;
+        NodeMetric metric = serverInfo.metric;
+        int period = serverInfo.metricPollPeriod;
+        serverInfo.refCount++;
+        if (serverInfo.refCount == Integer.MAX_VALUE) {
+            /* Reset to 1 when it is about to overflow */
+            serverInfo.refCountOnHeartBeat = serverInfo.refCount = 1;
+        }
+
+        try {
+            /* Forward time = Request Serialization + Network Delay + Deserialization + Object/call Dispatch time
```

Comment:
These is a weirdly formatted set of multi-line comments. Why not just one '/*' at the beginning, and one '*/' at the end?

Reply:
Formatted